front/app/components/ConsentManager/destinations.ts

```diff
@@ -12,13 +9,17 @@ export type TCategory = typeof CATEGORIES[number];
 
 const destinationConfigs: IDestinationConfig[] = [];
 
+export interface IConsentManagerFeatureMap {}
+export type TConsentManagerFeature =
+  IConsentManagerFeatureMap[keyof IConsentManagerFeatureMap];
+
 export interface IDestinationConfig {
   /** A unique key, used to name the destination in the stored cookie */
   key: IDestination;
   /** Destinations are grouped in categories. Under which category should it be listed? */
   category: TCategory | ((tenant: IAppConfigurationData) => TCategory);
   /** The name of the feature flag that should be active for the destination to be functional */
-  feature_flag?: keyof IAppConfigurationData['attributes']['settings'];
+  feature_flag?: TConsentManagerFeature;
```

Comment:
There was a type issue in destinations.ts, it checked enabled, but not all AppConfiguration settings have this now, so went with a pragmatic solution, explicitly defining the settings we need (which all have enabled).

Reply:
[This commit](https://github.com/CitizenLabDotCo/citizenlab/pull/2303/commits/8360783f7f43c9c0cde362fa980fbc68c3f5329a) has the complete thing.